core/src/main/c/share/dedup.cpp

```diff
@@ -667,6 +667,34 @@ Java_io_questdb_std_Vect_dedupMergeVarColumnLen(JNIEnv *env, jclass cl,
     return dst_var_offset;
 }
 
+
+JNIEXPORT jlong JNICALL
+Java_io_questdb_std_Vect_dedupMergeVarcharColumnLen(JNIEnv *env, jclass cl,
```

Comment:
this method should be called `dedupMergeVarcharColumnSize` for three reasons:
- everything that is called "size" in the driver provides size in bytes
- migrating to the driver i had very hard time establishing what is "size" and what is "row count", things were called "len" and "size" interchenageably and they could mean either row count or size depending on context

Reply:
Ok, done